viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -635,6 +639,32 @@ export class Cognite3DViewer {
     return model;
   }
 
+  /**
+   * Adds a set of 360 images to the scene from the /events API in Cognite Data Fusion.
+   * @param datasource The CDF data source which holds the references to the 360 image sets.
+   * @param eventFilter The metadata filter to apply when querying events that contains the 360 images.
+   * @param setTransform An optional addition transform which will be applied to all the 360 images in this set.
+   * @example
+   * ```js
+   * const eventFilter = { site_id: "12345" };
+   * await viewer.add360ImageSet(eventFilter);
+   * ```
+   */
+  add360ImageSet(
```

Comment:
I'd perhaps renamed the `eventFilter` parameter just so it makes sense as a filter for more than just the event datasource, when/if we eventually get another.

Reply:
Think it makes sense since it is a filter which only really makes sense for an event. When we overload this we can f.ex. add a new name `fdmFilter` or whatever we end up doing.